okhttp/src/main/java/okhttp3/EventListener.java

```diff
@@ -86,15 +86,8 @@ public void dnsStart(Call call, String domainName) {
    * Invoked immediately after a DNS lookup.
    *
    * <p>This method is always invoked after {@link #dnsStart(Call, String)}.
```

Comment:
nit: not true

Reply:
Fixed, `This method is invoked after {@link #dnsStart}`.